src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect, useState } from "react";
+import { Loader } from "../Loader";
+import { Person } from "../../types";
+import { Link, useParams } from "react-router-dom";
+import classNames from "classnames";
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  const { slug } = useParams();
+
+  useEffect(() => {
+    setLoading(true);
+
+    setTimeout(() => {
+      fetch('https://mate-academy.github.io/react_people-table/api/people.json', { method: 'GET' })
+        .then(respose => respose.json())
```

Comment:
Use getPeople function from api file
```suggestion
      getPeople()
```